mockup/patterns/resourceregistry/js/fields.js

```diff
@@ -366,6 +366,16 @@ define([
     }
   });
 
+
+  var MergeWithFieldView = ResourceSelectFieldView.extend({
+    multiple: false,
+    getSelectOptions: function() {
+      var self = this;
+      return ['', 'default', 'logged-in'];
```

Comment:
should this be a string field? or coming from a registry setting? thinking of other metabundles

Reply:
they are hardcoded anyways. see: https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/interfaces/resources.py#L119
I saved me the hassle to dynamize that which would involve making a proper vocabulary on the Plone site and so on...